doc/man3/SSL_export_keying_material.pod

```diff
@@ -13,14 +15,28 @@ SSL_export_keying_material - obtain keying material for application use
                                 const unsigned char *context,
                                 size_t contextlen, int use_context);
 
+ int SSL_export_keying_material_early(SSL *s, unsigned char *out, size_t olen,
+                                      const char *label, size_t llen,
+                                      const unsigned char *context,
+                                      size_t contextlen, int use_context);
+
 =head1 DESCRIPTION
 
 During the creation of a TLS or DTLS connection shared keying material is
 established between the two endpoints. The function SSL_export_keying_material()
-enables an application to use some of this keying material for its own purposes
-in accordance with RFC5705 (for TLSv1.2 and below) or RFCXXXX (for TLSv1.3).
+and SSL_export_keying_material_early() enables an application to use some of
+this keying material for its own purposes in accordance with RFC5705 (for
+TLSv1.2 and below) or RFCXXXX (for TLSv1.3).
 TODO(TLS1.3): Update the RFC number when the RFC is published.
 
+SSL_export_keying_material() derives keying material using
+exporter_master_secret established in handshake.
```

Comment:
...derives keying material using *the* exporter_master_secret established in *the* handshake.